src/components/calculator/PointsDisplay.tsx

```diff
@@ -46,6 +46,40 @@ export function ExplanationCard(props: {
             : ''}
         </b>
       </p>
+
```

Comment:
Please remove dead code.

Reply:
It's removed in #346. And I'm happy to remove it here too for consistency.